roles/deploy/tasks/main.yml

```diff
@@ -27,6 +27,11 @@
       repo: "{{ repo }}"
       dest: /home/{{ ansible_ssh_user }}/{{ project_name }}
 
+- name: install sqlite dependencies
```

Comment:
It seems like the newfia deploy worked without this.  would it be cleaner to change line 36 to:

shell: bundle install --without development test --deployment chdir=/home/{{ ansible_ssh_user }}/{{ project_name }}